types/client.d.ts

```diff
@@ -0,0 +1,19 @@
+import { Person } from "./person";
+import { Project } from "./project";
+
+export interface Client {
+    readonly id: string;
+    readonly name: string;
+    readonly website: string;
+    readonly accountManager: Person;
+    readonly clientRepresentative: Person;
+}
+
+export declare class ClientAPI {
+    getClients(): Array<Client>;

```

Comment:
If you change `getClients` to `listClients`, don't forget to update the remaining interfaces